constants.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+#include <vector>
+
+namespace PIECES {
+
+	const int NUMBER_OF_PIECES = 7;
+
+	const int INITIAL_ABS_POSITION_X = 5;
+
+	const int INITIAL_ABS_POSITION_Y = 20;
+
+	const int INITIAL_ROTATION = 0;
+
+	const int MOVE_UP = 1;
+
+	const int MOVE_DOWN = -1;
+
+	const int MOVE_LEFT = -1;
+
+	const int MOVE_RIGHT = 1;
+
+	const std::vector<int> SQUARE_REL_X_POSITIONS = { 0, 1, 0, 1 };
+
+	const std::vector<int> SQUARE_REL_Y_POSITIONS = { 0, 0, 1, 1 };
+
+	const std::vector<int> TBLOCK_REL_X_POSITIONS = { -1, 0, 1, 0, 0, 0, 0, 1, -1, 0, 1, 0, 0, 0, 0, -1 };
+
+	const std::vector<int> TBLOCK_REL_Y_POSITIONS = { 0, 0, 0, 1, 1, 0, -1, 0, 0, 0, 0, -1, 1, 0, -1, 0 };
+
+	const std::vector<int> LBLOCKL_REL_X_POSITIONS = { -1, 0, 1, 1, 0, 0, 0, 1, -1, 0, 1, -1, 0, 0, 0, -1 };
+
+	const std::vector<int> LBLOCKL_REL_Y_POSITIONS = { 0, 0, 0, 1, 1, 0, -1, -1, 0, 0, 0, -1, -1, 0, 1, 1 };
+
+	const std::vector<int> LBLOCKR_REL_X_POSITIONS = { -1, 0, 1, -1, 0, 0, 0, 1, -1, 0, 1, 1, 0, 0, 0, -1 };
+
+	const std::vector<int> LBLOCKR_REL_Y_POSITIONS = { 0, 0, 0, 1, 1, 0, -1, 1, 0, 0, 0, -1, -1, 0, 1, -1 };
+
+	const std::vector<int> STRAIGHT_REL_X_POSITIONS = { -1, 0, 1, 2, 1, 1, 1, 1, -1, 0, 1, 2, 0, 0, 0, 0 };
+
+	const std::vector<int> STRAIGHT_REL_Y_POSITIONS = { 0, 0, 0, 0, -1, 0, 1, 2, 1, 1, 1, 1, -1, 0, 1, 2 };
+
+	const std::vector<int> ZBLOCK_REL_X_POSITIONS = { -1, 0, 1, 2, 1, 1, 1, 1, -1, 0, 1, 2, 0, 0, 0, 0 };
+
+	const std::vector<int> ZBLOCK_REL_Y_POSITIONS = { 0, 0, 0, 0, -1, 0, 1, 2, -1, -1, -1, -1, -1, 0, 1, 2 };
+
+	const std::vector<int> SBLOCK_REL_X_POSITIONS = { -1, 0, 1, 2, 1, 1, 1, 1, -1, 0, 1, 2, 0, 0, 0, 0 };
+
+	const std::vector<int> SBLOCK_REL_Y_POSITIONS = { 0, 0, 0, 0, -1, 0, 1, 2, 1, 1, 1, 1, -1, 0, 1, 2 };
```

Comment:
nit: I dont like mixing X/Y and row/col naming schemes as it may lead to confusion later on.